flybywiresim/a32nx
# Prelim fix for uncontrollable dive

Preliminary fix for unrecoverable stalls leading to departure from controlled flight in CONF 1+F

Fixes #2661 

## Summary of Changes

 Changes the flight_model.cfg file to:
1. Comment out the automatic retraction of flap 1 from CONF 1+F when speed goes above 210 knots; and 
2. Fix the VFE speed for CONF 1+F (Changes it from 230 knots to 215 knots)
## Screenshots (if necessary)



## References




Reference for VFE change:
![Screenshot (463)](https://user-images.githubusercontent.com/70166617/103120995-ff43b780-462e-11eb-9a42-8eb7299a3372.png)

## Additional context

There have been numerous reports of the mod going into an unrecoverable dive after takeoff and during approach.  With the help of some videos, I have been able to reproduce the after takeoff issue. It occurs during a CONF 1+F takeoff where flaps are not retracted while the airplane speed increases beyond the speed at which the flap portion of CONF 1+F automatically retracts (210 knots) (although the upset does not occur until well after the flap retracts).

Removing the automatic flap retraction (or retracting to flaps up before 210 knots!) resolves the issue.  This is a preliminary fix to allow further work to pinpoint the exact cause and allow the automatic flap retraction to be reinstated as this is a feature of the real airplane. For that reason, I only commented out the automatic flap retraction line. I did not remove it.

Until I am able to reproduce the issue on approach, I don't know if this change will resolve the issue there.

The automatic flap retraction feature was not functioning 100% correctly anyway. It worked for taking off with CONF 1+F, but it should also work for retracting flaps in the air. When going from CONF 2 toward 1, .when at or below 210 knots it should go to 1+F, but above 210 knots, or if initially at or below 210 knots so it went to 1+F, when speed is increased to above 210 knots, it should go to 1. It does not.

I also corrected the VFE speed for CONF 1+F from 230 knots to 215 knots.

Discord username (if different from GitHub):

## Testing instructions

Do back-to-back testing with and without this PR to show you can reproduce the issue without it and that the issue does not occur with it. The easiest way to test (thanks to a video from @lukecologne):
1. Start on the runway (any runway should do)
2. Go to external view. No need to be distracted by any instruments, warnings,, CDU setup, or anything else. And you can see the stall indicator and watch the acrobatics better here.
3. Release parking brakes, go to TOGA thrust, rotate around 130 knots (not critical), and retract the landing gear with positive rate of climb.
4. Just keep a gentle climb with the primary purpose of allowing the airplane to accelerate.
5. Continue until the event occurs (probably around 270 KCAS) or doesn't occur (I've gone to 300 knots, which required a steadily decreasing pitch angle.)

Examples here:  https://1drv.ms/v/s!Ajsy7Ey4_8uFyBwIJwNwsxJ3gHNQ?e=YG3C9s (pre-PR) and https://1drv.ms/v/s!Ajsy7Ey4_8uFyB16ywtDFF4PNTAL?e=NPvM0t (post PR)

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
